plugins/auth-node/src/DefaultIdentityClient.test.ts

```diff
@@ -401,17 +401,29 @@ describe('DefaultIdentityClient', () => {
     it('given a corrupt the identity', async () => {
       await expect(
         client.getIdentity({
-          request: { headers: { authorization: `Bearer bad-token` } },
+          request: { headers: { authorization: 'Bearer bad-token' } },
```

Comment:
Im aware that I probably wrote this expectation. But I wonder is it possible to check the error type to make sure its an AuthenticationError?